conf/app.routes

```diff
@@ -205,6 +205,7 @@ GET         /my-organisation/claim/property-links/occupancy
 POST        /my-organisation/claim/property-links/occupancy                                   controllers.propertyLinking.ClaimPropertyOccupancyController.submitOccupancy
 GET         /my-organisation/claim/property-links/evidence                                    controllers.propertyLinking.ChooseEvidenceController.show
 POST        /my-organisation/claim/property-links/evidence                                    controllers.propertyLinking.ChooseEvidenceController.submit
+GET         /my-organisation/claim/property-links/occupier-evidence                           controllers.propertyLinking.ChooseEvidenceController.show
```

Comment:
did you mean to add this?

Reply:
Yeah, there was a bug that if you changed language on the occupier's evidence screen after getting an error it would display page not found

Reply:
oh yeah i see what you mean, thats awful 😬